manager_mq/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM rabbitmq:3.11-management-alpine
+
+RUN echo "consumer_timeout = 31622400000" >> /etc/rabbitmq/rabbitmq.conf
```

Comment:
Can you comment on why this is needed, and why it needs to be set to (apparently) 1 year?

Reply:
This is introduced because of the long time nmap can take and Celery timed out while nmap was still busy resulting in "PRECONDITION_FAILED and crashing Django. It should never crash Django and solved in the manager logic like now.

According to the Celery documentation in the WARNING section: '''To solve the problem, in RabbitMQ configuration file rabbitmq.conf you should specify the consumer_timeout parameter greater than or equal to your countdown value. For example, you can specify a very large value of consumer_timeout = 31622400000, which is equal to 1 year in milliseconds, to avoid problems in the future.'''